packages/amplify-codegen/src/commands/models.js

```diff
@@ -145,6 +146,13 @@ function getModelOutputPath(context) {
   }
 }
 
+function getAmplifyCLIVersionComment(context) {
+  if (context.usageData == null || !context.usageData.version) {
+    return null
+  }
+  return '// Generated using Amplify CLI version: ' + context.usageData.version;
```

Comment:
nit: in case we'll ever need/want to use this info programmatically, may be worth using something that is easier to parse like `// amplify-cli-version: X.Y.Z` (similar to what [SwiftPM](https://developer.apple.com/documentation/swift_packages/package) currently does)?

Reply:
Changed this to `// Generated using amplify-cli-version: X.Y.Z`. The "Generated" part serves the purpose of warning the user that these files are auto-generated. `amplify-cli-version: X.Y.Z` part of the comment should make parsing easier.